src/app.scss

```diff
@@ -1,3 +1,9 @@
+/*
+ * PF4 overrides need to come after the JSX components imports because
+ * these are importing CSS stylesheets that we are trying to override
```

Comment:
s/trying to override/overriding/

But this comment looks a bit weird here -- this file doesn't import any PF4 stylesheets. Is it possible to import app.scss in src/index.js instead? If not, let's move the comment to index.html in between importing pf-4-base.css and index.css? (although that's rather obvious even without the comment)

Reply:
patternfly-4-overrides is not overriding rules from patternfly-4-base.css :) It's fixing the stylesheets that are imported when we include React components form `@patternfly/react-core`.
patternfly-4-base.css is just the base so that the per-components stylesheets can be cleanly applied - imagine is as a CSS reset.

So, these fixes (patternfly-4-overrides.scss) have to be after all the PF4 components CSS is in the resulting dist/index.css produced file.
`mini-css-extract-plugin` does the work for us, since it takes all the CSS that was included in the dist/index.js files from the css/scss loaders and puts it in the dist/index.css in the order that these files where imported.
So after the scss/css loader run we do have a dist/index.js file full of CSS rules, in the order of PF4 components imports, app.scss compiled contents. Which is exactly what we want. 

However I understand the confusion - it's rather not clear - thus I moved the overrides in the index.js file but kept the app.scss where it was not make it clear that these are not connected.